### ISSUE TITLE
Add Clarifications tab to Shadow Frame

### ISSUE THREAD

**Issue by lane55** [pc2ccs/pc2v9#763]
**Is your feature request related to a problem?** 

It is helpful to be able to easily view clars while
monitoring shadowing.

**Feature Description**: 

Add the Clarifications tab to the Shadow Frame.

**Have you considered other ways to accomplish the same thing?** 

n/a

**Do you have any specific suggestions for how your feature would be ***implemented*** in PC^2?**

Do per description.

**Additional context**:

**Comment by lane55** [comment]
It is possible but unlikely that the Feeder is never sent clarifiation updates.   Very easy to test though.

**Comment by lane55** [comment]
When adding Clarifications Pane to feeder, the Answer button was present.
Remove ANSWER_CLARIFICATION Permission from default permissions for FEEDER account type.

**Comment by clevengr** [comment]
I don't understand the statement "Provides a way to review clars as Shadow".  Clarifications are submitted to the Primary CCS only.  The only way a PC2 Shadow would ever see a clarification is if the clar appeared, in complete form, in the Event Feed, AND we had a way to pull clars from such an Event Feed and "submit" them to the PC2 (Shadow) server.  Merely adding a Clars tab to the Shadow Feeder frame isn't going to accomplish that.  I see no point in just "adding a Clarifications tab to the Shadow Frame";  it appears to me that there is _much_ more work involved in making it so that a Shadow has the ability to see Clars.

**Comment by clevengr** [comment]
> > I don't understand the statement "Provides a way to review clars as Shadow". Clarifications are submitted to the Primary CCS only. The only way a PC2 Shadow would ever see a clarification is if the clar appeared, in complete form, in the Event Feed, AND we had a way to pull clars from such an Event Feed and "submit" them to the PC2 (Shadow) server. Merely adding a Clars tab to the Shadow Feeder frame isn't going to accomplish that. I see no point in just "adding a Clarifications tab to the Shadow Frame"; it appears to me that there is _much_ more work involved in making it so that a Shadow has the ability to see Clars.
> 
> [Issue #776](https://github.com/pc2ccs/pc2v9/issues/776)

Ok, I see that there is a separate issue (#776) for this.  That means that this issue (#763) should be marked in some way as "dependent on #776".  In Bugzilla there was a way to actually mark Issues as having dependencies.  I don't know if GitHub supports such a thing.
